packages/ComponentLibrary/src/components/Toggle/ToggleChip.tsx

```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+import Switch, { SwitchProps } from '@mui/material/Switch';
+import FormControlLabel from '@mui/material/FormControlLabel';
+import { styled, Theme } from '@mui/material/styles';
+import { switchStyles, formControlLabelStyles } from './toggle.styles';
+
+const StyledSwitch = styled(Switch)(({ theme }: { theme: Theme }) =>
+  switchStyles(theme),
+);
+
+interface ToggleChipProps extends SwitchProps {}
+
+const ToggleChip: React.FC<ToggleChipProps> = props => (
+  <FormControlLabel
+    control={<StyledSwitch {...props} />}
+    label=""
+    style={formControlLabelStyles}
+  />
+);
+
```

Comment:
It seems to me that the task is about being able to use a switch and not a labeled switch container, right?

Reply:
It's just for the chip component, do not introduce a formcontrol